elasticsearch/azuredeploy.json

```diff
@@ -295,23 +295,22 @@
     },
     "_artifactsLocation": {
       "type": "string",
-      "defaultValue": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/elasticsearch/",
+      "defaultValue": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/elasticsearch",
       "metadata": {
         "description": "Change this value to your repo name if deploying from a fork"
       }
     },
     "_artifactsLocationSasToken": {
       "type": "securestring",
-      "defaultValue": "?",
+      "defaultValue": "",
```

Comment:
@singhkay, unfortunately the question mark causes problems. For example: Windows CSE tries to download the target to a local file named "filename.ps1?" and the question mark is an invalid filename character. 
Was there a reason to use "?" rather than ""?